about.html

```diff
@@ -8,66 +8,102 @@
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/about.css">
 
 </head>
 
-<body>
-    <div class="container about-page">
-     
-    About
+  <body>
+      <div class="container">
+        <header>
+          <img class="logo" src="img/lambda-black.png" alt="Lambda logo">
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+          <nav>
+            <a href="index.html" target="_blank">Home</a>
+            <a href="about.html" target="_blank">About</a>
+            <a href="#">Products</a>
+            <a href="#">Blog</a>
+            <a href="contact.html" target="_blank">Contact</a>
+          </nav>
 
+        </header>    
+      
+        <img class ="HeaderIMG" src="img/jumbo-about.png">
```

Comment:
One thing to take note of is what you name you css classes. In this case it would be better to have had `class="header-img"` lowercase and a dash between words. You guys will learn about BEM naming conventions which is popular in the industry. What you have isn't wrong by any means, just conventional stuff 🙂